### DISCUSSION TITLE
Does the package support time dependent variables

### DISCUSSION THREAD

**Discussion by kewellcjj** [sebp/scikit-survival#275] [Q&A] [answered]
Thanks for the great package, haven't looked into the details. Wondering if time-dependent variables are supported by e.g. the counting process formulation?

**Comment by sebp** [comment, marked as answer]
Currently, time-dependent variables are not supported. See also https://github.com/sebp/scikit-survival/issues/8

**Reply by kewellcjj** [reply]
Thank you Seb, I follow #8 in the future.
